tests/sql/operators/test_load_table.py

```diff
@@ -0,0 +1,53 @@
+from unittest.mock import patch
+
+from airflow.models.connection import Connection
+
+from astro.sql.operators.load_table import LoadTableOperator
+
+
+@patch("sqlalchemy.create_engine")
+@patch("airflow.hooks.base.BaseHook.get_connection")
+def test_load_table_sqlite(mock_get_conn, mock_sqlalchemy_engine):
+    mock_get_conn.return_value = Connection(
+        conn_id="conn", conn_type="sqlite", host="localhost"
+    )
+    op = LoadTableOperator(
+        task_id="task1", sql_statement="select * from 1", conn_id="conn"
+    )
+    op.execute(None)
+    mock_sqlalchemy_engine.assert_called_once_with("sqlite:///localhost")
+
+
+@patch("astro.databases.base.BaseDatabase.sqlalchemy_engine")
+@patch("airflow.hooks.base.BaseHook.get_connection")
+def test_load_table_postgres(mock_get_conn, sqlalchemy_engine):
+    mock_get_conn.return_value = Connection(
+        conn_id="conn", conn_type="postgres", host="localhost"
+    )
+    op = LoadTableOperator(
+        task_id="task1", sql_statement="select * from 1", conn_id="conn"
+    )
+    op.execute(None)
+
+
+@patch("astro.databases.google.bigquery.BigqueryDatabase.sqlalchemy_engine")
+@patch("airflow.hooks.base.BaseHook.get_connection")
+def test_load_table_bigquery(mock_get_conn, mock_sqlalchemy_engine):
+    mock_get_conn.return_value = Connection(conn_id="conn", conn_type="gcpbigquery")
```

Comment:
Whats this test doing? Its not asserting anything.

Reply:
fixed it